ui/job-view/pushes/Push.jsx

```diff
@@ -245,19 +234,19 @@ class Push extends React.Component {
         (platformArray.indexOf(b.name) * 100 + (thOptionOrder[b.option] || 10)),
     );
     return platforms;
-  }
+  };
 
-  expandAllPushGroups(callback) {
+  expandAllPushGroups = callback => {
     // This sets the group state once, then unsets it in the callback.  This
     // has the result of triggering an expand on all the groups, but then
     // gives control back to each group to decide to expand or not.
     this.setState({ pushGroupState: 'expanded' }, () => {
       this.setState({ pushGroupState: 'collapsed' });
       callback();
     });
-  }
+  };
 
-  showUpdateNotifications(prevState) {
+  showUpdateNotifications = prevState => {
```

Comment:
It wasn't, that's true.  To be honest it looked like it SHOULD have needed it.  But perhaps because it is only called from within a lifecycle method, it didn't need it?  I haven't yet found an explanation online.  But this works well and with our conversation in IRC, I'll go ahead an err on the side of more functions in this format.